packages/gasket-plugin-manifest/package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Are we wanting to start committing package-lock files?

Reply:
No, we want to avoid, but currently, the manifest plugin depends on `mkdirp@^1.0.4`, where everything else is on `mkdirp@^0.5.1`. I'm not sure why a package-lock wasn't generated before...

 It's a code change to line things up. I'm going to add a ticket for this.